scarb/src/bin/scarb/args.rs

```diff
@@ -341,6 +342,8 @@ pub struct FmtArgs {
     /// Specify package(s) to format.
     #[command(flatten)]
     pub packages_filter: PackagesFilter,
+    #[clap(value_name = "TARGET")]
+    pub target: Option<PathBuf>,
```

Comment:
```suggestion
    pub target: Option<Utf8PathBuf>,
```
Please use `Utf8PathBuf` instead